src/directory.rs

```diff
@@ -20,9 +17,7 @@ pub(crate) fn get_file_iter(
             match mimetype::image_or_video(&f_path) {
                 Ok(true) => v.push(PathBuf::from(f_path)),
                 Ok(false) => {}
-                Err(_) => {
-                    eprintln!("skip {}", &f_path.display())
-                }
+                Err(_) => eprintln!("skip {}", &f_path.display()),
```

Comment:
I would prefer if this would be an error logging and including the path and the error. What do you think about this? The  way it is now it is user output, is this on purpose?